### DISCUSSION TITLE
Plus button's dropdown is always visible

### DISCUSSION THREAD

**Discussion by mr-cheffy** [community/community#57030] [Codespaces]
### Select Topic Area

Bug

### Body

This bug only occurs in laptops. The `add` button's dialogue is always open since the page is loaded until I exit it. No matter where I click, it's always available.

![image](https://github.com/community/community/assets/91018726/e2ef98d3-244f-431e-8f5d-37059d11db95)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @mauro-balades! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by mr-cheffy** [reply]
Thanks! But this isn't codespace specific, it's more about the UI
